src/content/api/cli.md

```diff
@@ -133,55 +134,55 @@ webpack --env.platform=web  # sets env.platform == "web"
-`--config-register, -r`   | Preload one or more modules before loading the webpack configuration | array | 
-`--config-name`           | Name of the config to use                   | string     | 
-`--env`                   | Environment passed to the config, when it is a function | 
-`--mode`                  | Mode to use, either "development" or "production" | string      | 
+| Parameter               | Explanation                                                          | Input type | Default                             |
+| ----------------------- | -------------------------------------------------------------------- | ---------- | ----------------------------------- |
+| `--config`              | Path to the config file                                              | string     | webpack.config.js or webpackfile.js |
+| `--config-register, -r` | Preload one or more modules before loading the webpack configuration | array      |
+| `--config-name`         | Name of the config to use                                            | string     |
+| `--env`                 | Environment passed to the config, when it is a function              |
+| `--mode`                | Mode to use, either "development" or "production"                    | string     |
 
 ### Output Options
 
 This set of options allows you to manipulate certain [output](/configuration/output) parameters of your build.
 
-Parameter                 | Explanation                                 | Input type | Default
-------------------------- | ------------------------------------------- | ---------- | ------------------
-`--output-chunk-filename` | The output filename for additional chunks   | string     | filename with [id] instead of [name] or [id] prefixed
-`--output-filename`       | The output filename of the bundle           | string     | [name].js
-`--output-jsonp-function` | The name of the JSONP function used for chunk loading | string | webpackJsonp
-`--output-library`        | Expose the exports of the entry point as library | string |
-`--output-library-target` | The type for exposing the exports of the entry point as library | string | var
-`--output-path`           | The output path for compilation assets      | string     | Current directory
-`--output-pathinfo`       | Include a comment with the request for every dependency | boolean | false
-`--output-public-path`    | The public path for the assets              | string     | /
-`--output-source-map-filename` | The output filename for the SourceMap  | string     | [name].map or [outputFilename].map
-`--build-delimiter` | Display custom text after build output | string | Default string is null. You could provide a string such as `=== Build done ===`
+| Parameter                      | Explanation                                                     | Input type | Default                                                                         |
```

Comment:
I still see lots of table changes though, did you accidentally miss these and the rest of table changes?

Reply:
yep, my bad, diff was collapsed. gonna address this today sorry :)

Reply:
hey @EugeneHlushko, reverted those missing changes, should be ok now